src/node.cc

```diff
@@ -877,8 +877,7 @@ Local<Value> ErrnoException(Isolate* isolate,
 
   Local<String> path_string;
   if (path != nullptr) {
-    // FIXME(bnoordhuis) It's questionable to interpret the file path as UTF-8.
-    path_string = String::NewFromUtf8(env->isolate(), path);
+    path_string = FIXED_ONE_BYTE_STRING(env->isolate(), path);
```

Comment:
It's still an improvement of sorts.  Any byte string is a valid Latin-1 string (although not per se a meaningful Latin-1 string) whereas conversion to UTF-8 is lossy - it's not bidirectional when the byte string contains invalid UTF-8 character sequences because those get replaced with U+FFFD.